qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/SnapshotReplicationTest.java

```diff
@@ -28,7 +29,7 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.zip.CRC32;
+import java.util.zip.CRC32C;
```

Comment:
:laughing: :+1:

Reply:
We can also use that in the replication chunks right?

Reply:
Yes - and also do a checksum of the full snapshot. I'll write an issue for it.